Cubical/Categories/Displayed/Presheaf/Morphism.agda

```diff
@@ -730,6 +723,57 @@ module _ {C : Category ℓC ℓC'}{Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'}
             ∙ Qᴰ.≡in (αᴰ .N-homᴰ)
             ∙ Qᴰ.⟨⟩⋆⟨ Qᴰ.reind-filler _ _ ⟩
 
+module _ {C : Category ℓC ℓC'}{Cᴰ : Categoryᴰ C ℓCᴰ ℓCᴰ'}
+  {P : Presheaf C ℓP}{Q : Presheaf C ℓQ}
+  {α : PshIso P Q} {Qᴰ : Presheafᴰ Q Cᴰ ℓQᴰ}
+  where
+
+  private
+    module C = Category C
+    module Cᴰ = Categoryᴰ Cᴰ
+    module Qᴰ = PresheafᴰNotation Qᴰ
+    module P = PresheafNotation P
+    module Q = PresheafNotation Q
+
+  module _ {Pᴰ : Presheafᴰ P Cᴰ ℓPᴰ} (αᴰ : PshIsoᴰ α Pᴰ Qᴰ) where
+    private
+      module Pᴰ = PresheafᴰNotation Pᴰ
+      PshHomEqPshIsoᴰ-isPshIso :
+        (αβ-N-ob : αβ-N-ob-ty {α = α .trans}) →
+        ∀ x → Σ[ g ∈ (Q.p[ x ] → P.p[ x ]) ]
+              Σ[ _ ∈ section (αβ-N-ob .fst x) g ]
+              retract (αβ-N-ob .fst x) g
+      PshHomEqPshIsoᴰ-isPshIso (_ , Eq.refl) = α .nIso
+
+      PshHomEqPshIsoᴰ-isPshIsoᴰ :
+        (αβ-N-ob : αβ-N-ob-ty {α = α .trans}) →
+          ∀ {x}{xᴰ : Cᴰ.ob[ x ]} →
+          isIsoOver
+            (isIsoToIso (PshHomEqPshIsoᴰ-isPshIso αβ-N-ob x))
+            Pᴰ.p[_][ xᴰ ]
+            Qᴰ.p[_][ xᴰ ]
+            (λ _ → PshHomEqPshHomᴰ-N-obᴰ (αᴰ .fst) αβ-N-ob)
+      PshHomEqPshIsoᴰ-isPshIsoᴰ (_ , Eq.refl) = αᴰ .snd
+
+    module _ {β : PshIso P Q} (eq-N-ob : α .trans .N-ob Eq.≡ β .trans .N-ob) where
+      private
+        isPshIso≡ :
+          PshHomEqPshIsoᴰ-isPshIso (_ , eq-N-ob) ≡ β .nIso
+        isPshIso≡  =
+            isPropIsPshIso {α = β .trans}
+              (PshHomEqPshIsoᴰ-isPshIso (_ , eq-N-ob)) (β .nIso)
+
+      PshHomEqPshIsoᴰ : PshIsoᴰ β Pᴰ Qᴰ
+      PshHomEqPshIsoᴰ .fst = PshHomEqPshHomᴰ (αᴰ .fst) eq-N-ob
+      PshHomEqPshIsoᴰ .snd =
+        subst
```

Comment:
this is a bad `subst`, I think instead of doing a `subst` here you should just take in a `eq-N-ob⁻ : ∀ x → α .nIso x .fst Eq.≡ β .nIso x .fst`

Reply:
don't bother unless you need it locally. I have a less manual way of doing this anyway